README.md

```diff
@@ -85,21 +84,18 @@ OneUI comes with a Storybook of examples for all components. [Click here](https:
 In order to run it yourself locally...
 
 1. Make sure you have Storybook installed (globally):
+
 ```bash
 $ npm i -g @storybook/cli
 ```
+
 2. Within the OneUI root, run `npm run storybook`
 3. Go to http://localhost:9001 to check out examples of all OneUI components
 
-## Example of implementation
+## Example implementation
 
 An implementation example can be found in the example directory, along with instructions on how to run it.
 
-## Testing
```

Comment:
Is this no longer relevant? I can't see how OneUI tests are to be ran (apart from testing the application that implements OneUI)

Reply:
This section is not relevant for README but for CONTRIBUTE. However just to list the scripts that are in package.json is quite redundant. If you think it has actual value, I can put it in CONTRIBUTE.md#Testing

Reply:
Ok agree, if you want you can still add it to the paragraph you mention.